src/config/dynamic-theme-fixes.config

```diff
@@ -5106,12 +5106,14 @@ img[src*="/static/media/logo-large"]
 img[src*="/static/media/samsung"]
 img[src*="/static/media/sonos"]
 .d-md-inline > img
-g.rsm-geographies
 
 CSS
 .text-right[style*="opacity: 0.3"] {
     opacity: 0.6 !important;
 }
+g.rsm-geographies {
+    filter: invert(1) hue-rotate(180deg) brightness(130%) !important;
```

Comment:
I believe 130% brightness looks good.
![firefox (2020-12-02) (09-47-34 PM)](https://user-images.githubusercontent.com/14346945/100923642-256eaf80-34e8-11eb-8b36-d06d0197c3df.jpg)

There's still room for improvement though, cause changing dark reader's settings slightly makes it look bad.

Reply:
Ideally all fixes are made for default settings.

Reply:
I see, thanks for your help!